.github/workflows/build.yml

```diff
@@ -183,6 +183,12 @@ jobs:
       CCACHE_NOCOMPRESS: 1
       MAKEFLAGS: -j2
     steps:
+      - name: enable POSIX MinGW
```

Comment:
```suggestion
      - name: Enable POSIX MinGW
```

Reply:
@sakertooth amended, thanks for catching this.